src/main/java/org/threeten/extra/scale/UtcRules.java

```diff
@@ -226,6 +226,70 @@ public TaiInstant convertToTai(UtcInstant utcInstant) {
      */
     public abstract UtcInstant convertToUtc(TaiInstant taiInstant);
 
+    /**
+     * Converts a {@code MispInstant} to a {@code UtcInstant}.
+     * <p>
+     * This method converts from the MISP time scale to the UTC time scale using the
+     * leap-second rules of the implementation.
+     *
+     * @param mispInstant  the MISP instant to convert, not null
+     * @return the converted UTC instant, not null
+     * @throws DateTimeException if the valid range is exceeded
+     * @throws ArithmeticException if numeric overflow occurs
+     */
+    public abstract UtcInstant convertToUtc(MispInstant mispInstant);
+
+    //-----------------------------------------------------------------------
+    /**
+     * Converts a {@code UtcInstant} to a {@code MispInstant}.
+     * <p>
+     * This method converts from the UTC to the MISP time-scale using the
+     * leap-second rules of the implementation.
+     *
+     * @param utcInstant  the UTC instant to convert, not null
+     * @return the converted MISP instant, not null
+     * @throws DateTimeException if the valid range is exceeded
+     * @throws ArithmeticException if numeric overflow occurs
+     */
+    public MispInstant convertToMISP(UtcInstant utcInstant) {
```

Comment:
Change MISP to Misp everywhere

Reply:
Have applied (in method names, not in documentation - assume that was the intent).